user_service/apis/handlers/user.go

```diff
@@ -12,10 +13,12 @@ import (
 
 type IUserHandlers interface {
 	HandleGetProfile(ctx *gin.Context)
+	HandleGetUserEmailsByIDs(ctx *gin.Context)
```

Comment:
`GetUserByIDs()` is a more generic approach. Return the basic user info off all the user-ids.